gpt4all-chat/modellist.cpp

```diff
@@ -403,11 +406,9 @@ bool DownloadableModels::filterAcceptsRow(int sourceRow,
     // FIXME We can eliminate the 'expanded' code as the UI no longer uses this
     bool withinLimit = sourceRow < (m_expanded ? sourceModel()->rowCount() : m_limit);
     QModelIndex index = sourceModel()->index(sourceRow, 0, sourceParent);
-    bool isDownloadable = !sourceModel()->data(index, ModelList::DescriptionRole).toString().isEmpty();
-    bool isInstalled = sourceModel()->data(index, ModelList::InstalledRole).toBool();
-    bool isIncomplete = sourceModel()->data(index, ModelList::IncompleteRole).toBool();
+    bool hasDescription = !sourceModel()->data(index, ModelList::DescriptionRole).toString().isEmpty();
     bool isClone = sourceModel()->data(index, ModelList::IsCloneRole).toBool();
-    return withinLimit && !isClone && !isInstalled && (isDownloadable || isIncomplete);
+    return withinLimit && hasDescription && !isClone;
```

Comment:
To be clear, they still show up on the downloadable models list too?

Reply:
Yes, they still appear where they did in the downloadable models list.

If you start downloading a model from HF and then clear your search results, it disappears from the download list as before, but you now you can find the download progress indicator in the "Installed Models" page instead of nowhere at all.